arlo_server/routes.py

```diff
@@ -1302,7 +1304,7 @@ def jurisdictionadmin_login_callback():
 @app.route("/")
 @app.route("/election/<election_id>")
 @app.route("/election/<election_id>/board/<board_id>")
-def serve(election_id=None, board_id=None):
+def serve(election_id=None, board_id=None):  # pylint: disable=unused-argument
```

Comment:
any reason not to just remove these args?

Reply:
I'm not positive, but I think that it'll crash if we do that:

```py
>>> def fn():
...   pass
... 
>>> fn(election_id='abc')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: fn() got an unexpected keyword argument 'election_id'
```